client/my-sites/domains/domain-management/edit/style.scss

```diff
@@ -38,3 +38,55 @@
 		}
 	}
 }
+
+@mixin domain_status_color( $color ) {
+	border-top: 5px solid $color;
+	> div.domain-status__icon {
+		color: $color;
+		> svg {
+			fill: $color;
+		}
+	}
+}
+
+.domain-types__container {
+	.status-success {
+		@include domain_status_color( #008a20 );
+	}
+	.status-pending {
+		@include domain_status_color( #3582C4 );
+	}
+	.status-warning {
+		@include domain_status_color( #B69000 );
+	}
+	.status-error {
+		@include domain_status_color( #C9356E );
+	}
+
+	.domain-status__card {
+		h2 {
+			font-size: 32px;
+		}
+
+		> .domain-status__icon {
+			font-size: 16px;
```

Comment:
I know I have the size set to 16px in my design but we should remove this so it defaults to the body font size which I think is 15px. There was a proposal to update our type sizes that was implemented in our Figma libraries but never in code.